tests/FSharpLint.Core.Tests/Rules/Binding/FavourIgnoreOverLetWild.fs

```diff
@@ -62,3 +62,44 @@ module Program
         let result = this.ApplyQuickFix source
 
         Assert.AreEqual(expected, result)
+
+    [<Test>]
+    member this.LetWildcardMultilaneStatementOfUnitSuggestedFix() =
+        let source = """
+module Program
+
+let (_) =
+  let x = 4 + 4
+  ()"""
+        let expected = """
+module Program
+
+(let x = 4 + 4
+  ()) |> ignore"""
```

Comment:
This code will not compile. And the whole test is kinda silly IMHO.

Reply:
agreed, the testcase doesn't make any sense